Sources/ScipioKit/BuildOptions.swift

```diff
@@ -33,6 +72,10 @@ struct BuildOptions: Hashable, Codable, Sendable {
     /// - Note: It have to store the actual file contents rather than its path,
     /// because the cache key should change when the file contents change.
     let customFrameworkModuleMapContents: Data?
+
+    #if compiler(>=6.0)
+    private(set) var environment: Environment?
```

Comment:
> Cannot automatically synthesize `Decodable` because `environment` does not have a matching CodingKey and does not have a default value.

The reason I use `var` in combination with a private setter is to avoid the error above. So to exclude `environment` from the `codable` process we have to:
- use the custom coding keys with a private setter or 
- implement a custom `Encodable/Decodable` conformance (which would be even more code).

Let me know if you see an alternative approach.